common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -532,6 +535,14 @@
     public static final String E_GET_ALL_SKINS_REQUEST = "GetAllSkinsRequest";
     public static final String E_GET_ALL_SKINS_RESPONSE = "GetAllSkinsResponse";
 
+    //ABQ
+    public static final String E_ABQ_DEVICE_REQUEST = "ABQDeviceRequest";
+    public static final String E_ABQ_DEVICE_RESPONSE = "ABQDeviceResponse";
+    public static final String E_DEVICE_ID = "deviceId";
```

Comment:
Can we rename the constant as E_ABQ_DEVICE_ID

Reply:
done